src/Mews.Fiscalization.Greece/RestClient.cs

```diff
@@ -48,75 +48,91 @@ internal RestClient(string userId, string subscriptionKey, AadeEnvironment envir
             HttpClient.DefaultRequestHeaders.Add(SubscriptionKeyHeaderName, $"{SubscriptionKey}");
         }
 
-        internal async Task<ResponseDoc> SendRequestAsync(InvoicesDoc invoicesDoc)
+        internal Task<ResponseDoc> SendRequestAsync(InvoicesDoc invoicesDoc)
         {
             var requestContent = XmlManipulator.Serialize(invoicesDoc).DocumentElement.OuterXml;
             Logger?.Debug("Created XML document from DTOs.", new { XmlString = requestContent });
 
             var requestMessage = BuildHttpPostMessage(requestContent);
+            
+            return SendAsync(
+                sendFunc:
+                    () => HttpClient.SendAsync(requestMessage),
+                buildErrorResponseFunc:
+                    (errorCode, errorMessage) => BuildResponseDocWithErrors(errorCode, errorMessage, invoicesDoc.Invoices),
+                buildResponseFunc:
+                    async httpResponseMessage =>
+                    {
```

Comment:
We generally don't do an extra indent on next line, so this would be just 

```suggestion
                sendFunc: () => HttpClient.SendAsync(requestMessage),
                buildErrorResponseFunc: (errorCode, errorMessage) => BuildResponseDocWithErrors(errorCode, errorMessage, invoicesDoc.Invoices),
                buildResponseFunc: async httpResponseMessage =>
                {
```
(without the last indent on the brackets as well.)

Reply:
Will fix this in the next PR, thanks